services/QuillLMS/spec/factories/email_subscriptions.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: email_subscriptions
+#
+#  id                :bigint           not null, primary key
+#  cancel_token      :string           not null
+#  frequency         :string           not null
+#  params            :jsonb
+#  subscription_type :string           not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  user_id           :integer          not null
+#
+FactoryBot.define do
+  factory :email_subscription do
+    frequency { EmailSubscription::MONTHLY }
+    subscription_type { EmailSubscription::ADMIN_DIAGNOSTIC_REPORT }
+    association :user
```

Comment:
I tend to prefer the shorter: `user`
I only use `association` when it's a non-standard factory that I need to refer to.

Reply:
Seems reasonable to me.  I'll make this change.